commit 769796ed4c852a2b9475c126925eeeffc215ce45
Merge: abf0693 e8c92e0
Author: ErikZalm <erik@vdzalm.eu>
Date:   Tue Sep 11 02:10:39 2012 -0700

    Merge pull request #240 from ZetaPhoenix/Marlin_v1_bugfix
    
    Missing #else in temperature.cpp prevented reading bed temp when TEMP_BE...

diff --git a/Marlin/temperature.cpp b/Marlin/temperature.cpp
index 7307b24..a0413f7 100644
--- a/Marlin/temperature.cpp
+++ b/Marlin/temperature.cpp
@@ -885,6 +885,8 @@ ISR(TIMER0_COMPB_vect)
       #if (TEMP_BED_PIN > -1)
         #if TEMP_BED_PIN > 7
           ADCSRB = 1<<MUX5;
+        #else
+          ADCSRB = 0;
         #endif
         ADMUX = ((1 << REFS0) | (TEMP_BED_PIN & 0x07));
         ADCSRA |= 1<<ADSC; // Start conversion
